src/fluids.md

```diff
@@ -0,0 +1,184 @@
+
+### Pressure outlet
+
+If an outlet of a domain is supposed to represent a pressure outlet (e.g.
+zero-pressure), one should use the method *NavierSolver::AddPresDirichletBC*.
+This enforces the pressure value $p^{n+1}$ in \eqref{eq:prespois}.
+
+### Zero-stress
+
+This boundary condition is used to represent an outflow attribute. Due to the
+nature of the $H^1$ finite-element discretization, the terms arise naturally in
+\eqref{eq:prespois} and \eqref{eq:hlm} resulting in
+
+\begin{align}
+    \nu \nabla u \cdot \hat{n} - p \cdot \hat{n} = 0.
+\end{align}
+
+If there is no other boundary condition applied to a certain attribute, this
+boundary condition is applied automatically (not through modification but rather
+through the formulation).
+
+## Solvers and preconditioners
+
+The choice of solvers and preconditioners for \eqref{eq:prespois} and
+\eqref{eq:hlm} are essential for the performance and robustness of the
+simulation.
+
+The pressure Poisson equation \eqref{eq:prespois} is solved using the CG Krylov
+method in combination with the low-order refined preconditioning technique
+coupled with AMG (c.f. Franco et al (2020)[^2]).
+
+Due to the nature of the explicit time discretization of the nonlinear term, the
+method used is CFL (and therefore time step) bound. As a result the time
+derivative term in \eqref{eq:hlm} is dominating and a CG Krylov method
+preconditioned with Jacobi is sufficient.
+
+Depending on the problem, this results in the majority of time per time step is
+spent in the pressure Poisson solve.
+
+At the moment there is no interface to change the default options for the
+solvers, but a user can easily modify them in the code itself.
+
+## FAQ
+
+- **You are using the spectral element method, why is the mass matrix not a
+vector representing the condensed diagonal?** This is a design choice. It is
+possible to use the "numerical integration" option, which produces a diagonal
+mass matrix with 1 non zero value per row. This leaves freedom to experiment.
+
+- **Do you support simulations using real parameters?** No, right now you have
+ to non-dimensionalize your problem. Not doing this impacts the performance a
+ lot.
+
+- **I want to implement turbulence model X, how do I dot that?** This is another
+ design choice to make and should be discussed, preferably in a Github issue.
+
+- **Why doesn't it have adaptive time stepping?** While it is possible and there
+exists a branch that works with varying step sizes (variable order/variable step
+size IMEX), I have not found a reliable and robust method to determine the step
+size (CFL based error estimators are very squishy here or have to use a very
+conservative limit).
+
+- **How do I compute steady state solutions with this?** There is no
+ acceleration to steady state algorithm implemented right now. Your only option
+ is to run the transient case until you reach a steady state criterion. (See
+ adaptive time stepping problematic).
```

Comment:
I'm not sure what is meant by "adaptive time stepping problematic" or where to find it.

Reply:
I was referring to the FAQ right above this one.

Reply:
I suspected that but it wasn't obvious because the wording was rather different.

Reply:
I changed the wording a bit to make this more clear.